File: java/Car.java
/**
 * Car
 */
public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver) {
        super();
        this.license = license;
        this.driver = driver;

    }

    public void printDataCar() {
        System.out.println("License" + license + "Your driver name : " + driver.name);
    }
}

File: JS/Car.js
class Car {
  constructor(license, driver, passenger) {
    this.id
    this.license = license;
    this.driver = driver
    this.passenger = passenger
  }

  printDataCar() {
    console.log(this.driver);
    console.log(this.driver.name);
    console.log(this.driver.document);
    console.log(`El numero de pasajeros es ${this.passenger}`);
  }
}